src/components/dialogs/network-modifications/tabular-modification/tabular-modification-dialog.js

```diff
@@ -72,6 +75,16 @@ const TabularModificationDialog = ({
 
     const { reset } = formMethods;
 
+    const toEnumValue = useCallback(
```

Comment:
You can remove enumValues arg and use ENERGY_SOURCES in this method.
And rename method to getEnergySourceId(label)

Reply:
This method is intended to be also used for LOAD_TYPES